testcontainers-dapr/src/main/java/io/dapr/testcontainers/AppHttpPipeline.java

```diff
@@ -0,0 +1,20 @@
+package io.dapr.testcontainers;
+
+import java.util.Collections;
+import java.util.List;
+
+public class AppHttpPipeline implements ConfigurationSettings {
+  private List<ListEntry> handlers;
+
+  public AppHttpPipeline() {
+  }
```

Comment:
WDYT to init an empty handlers list here?

Reply:
+1 - shall we initialize to empty `ArrayList` or similar here to prevent potential issues if clients try to access this before setting it?

Reply:
I don't even think we need that empty constructor to be honest.